include/gatekeeper_net.h

```diff
@@ -32,6 +34,7 @@ struct net_config {
 	uint32_t		num_ports;
 	uint32_t		numa_nodes;
 	struct rte_mempool 	**gatekeeper_pktmbuf_pool;
+	uint16_t		queues[RTE_MAX_LCORE];
```

Comment:
This allocation should be per interface, and per rx and tx. That is, we need four arrays.

The motivation for this is that not all blocks are going to need a single queue on both devices, and the type of those queues may be different. The GK-GT unit, for example, requires no queue from the front interface, and two queues from the back interface.